roamance/src/api/roamance-api.ts

```
@@ -1,21 +1,186 @@
+        }
+
+        // Add token to request headers if available
+        const token = this.getToken();
+        if (token && config.headers) {
+          // Ensure the token is properly formatted
+          config.headers.Authorization = `Bearer ${token}`;
+          // For debugging
+          console.log(
+            'Request with token:',
+            config.url,
+            token.substring(0, 10) + '...'
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)